src/components/Chat/ChatHeaderComponent.tsx

```diff
@@ -33,22 +32,19 @@ const ChatHeaderComponent = (props: IProps) => {
         <span className="chat-header--text--right">
           <p>{course}</p>
           <button
-            onClick={() => setIsModalOpen(true)}
+            onClick={() => setIsOpen(true)}
             className="leksehjelp--button-success"
           >
             Forlat Chatten
           </button>
         </span>
       </div>
-      {isModalOpen && (
-        <Modal
-          content="Er du sikker på at du vil forlate chaten?"
-          warningButtonText="Forlat Chatten"
-          warningCallback={leaveChat}
-          successButtonText="Bli i Chatten"
-          handleClose={handleCloseModal}
-        />
-      )}
+      <Modal
+        content="Er du sikker på at du vil forlate chaten?"
```

Comment:
```suggestion
        content="Er du sikker på at du vil forlate chatten?"
```
Bli enig om det er stor eller liten bokstav også, ref linje 45 og 46